python/langsmith/client.py

```diff
@@ -3725,11 +3727,20 @@ def list_examples(
             params["dataset"] = dataset_id
         else:
             pass
+        if get_attachment_urls == True:
+            params['select'] = ['attachment_urls', 'outputs'] # don't need to add other stuff, because of always_select variable in API endpoint
         for i, example in enumerate(
             self._get_paginated_list("/examples", params=params)
         ):
+            attachment_urls = {}
+            if example['attachment_urls']:
+                for key, value in example['attachment_urls'].items():
+                    attachment_urls[key.split(".")[1]] = value['presigned_url']
```

Comment:
what's the point of doing this? also need to fail gracefully if [1] causes an index out of bounds error

Reply:
All the attachments appear as "attachment.name", can keep as is if you want (i.e. don't do the splitting and always include the "attachment." prefix.

Reply:
I believe this is from `ATTACHMENT_S3_PREFIX = "attachment."` line in `ingest.py`